### ISSUE TITLE
typo in readme and the example doesn't work

### ISSUE THREAD

**Issue by wildwestrom** [rasendubi/uniorg#12]
File: `uniorg/packages/uniorg-extract-keywords/readme.md`

The examples in the readme was incorrect. Took me ages to figure out what went wrong.

```diff
const unified = require('unified')
- const toVfile = require('to-vfile')
+ const toVFile = require('to-vfile')
const uniorgParse = require('uniorg-parse')
const extractKeywords = require('uniorg-extract-keywords')
const uniorg2rehype = require('uniorg-rehype')
const html = require('rehype-stringify')

unified()
  .use(uniorgParse)
  .use(extractKeywords)
  .use(uniorg2rehype)
  .use(html)
-  .process(toVfile.readSync('./example.org'), function (err, file) {
+  .process(toVFile.readSync('./example.org'), function (err, file) {
    console.log(file.toString());
    console.log(file.data);
  })
  ```
  --- 
  Also, perhaps more importantly, the example outputs the wrong text.
Expected:
```
<p>other org-mode</p>

{ title: 'Post title', author: 'Your Name' }
```
Actual:
```
#+TITLE: Post title
#+AUTHOR: Your Name

other org-mode

{}
```

Is this because I'm running it wrong?
I ran the following:
```shell
node app.js
```

Perhaps you may also want to use `import` as opposed to `require()`.
```js
import unified from 'unified';
import { toVFile } from 'to-vfile';
import uniorgParse from 'uniorg-parse';
import extractKeywords from 'uniorg-extract-keywords';
import uniorg2rehype from 'uniorg-rehype';
import html from 'rehype-stringify';

unified()
    .use(uniorgParse)
    .use(extractKeywords)
    .use(uniorg2rehype)
    .use(html)
    .process(toVFile.readSync('../src/posts/example.org'), function(err, file) {
        console.log(file.toString());
        console.log(file.data);
    })
```

**Comment by wildwestrom** [comment]
whoops, wrong repo

**Comment by wildwestrom** [comment]
Also, perhaps more importantly, the example outputs the wrong text.
Expected:
```
<p>other org-mode</p>

{ title: 'Post title', author: 'Your Name' }
```
Actual:
```
#+TITLE: Post title
#+AUTHOR: Your Name

other org-mode

{}
```

Is this because I'm running it wrong?
I ran the following:
```shell
node app.js
```

**Comment by wildwestrom** [comment]
Just quoting from PR #13.

@rasendubi:
> This looks like a breaking change in to-vfile@7. (unified ecosystem seems to migrate to ESM-only modules, so that's why `require`s stop working.)
> 
> Thanks for the report and fix! I'll look into why the example produces the wrong result.

Thank you very much, please do.
I'm using this in one of my own projects and your parser is super accurate! I could perhaps build a docker image with emacs to help with CI, but if `extract-keywords` works, this would help greatly. Cheers!

**Comment by rasendubi** [comment]
`uniorg-extract-keywords` must be working as it passes all tests, works in org-braindump and next-blog-starter [examples](https://github.com/rasendubi/uniorg/tree/master/examples), and I also use it for my website. Must be something wrong with the example…

**Comment by rasendubi** [comment]
ok, found another breaking change in vfile@5: `contents` field has been renamed to `value`. to-vfile@7 pulls in vfile@5, but unified@9 depends on vfile@4, so this leads to two versions of vfile to be present:
```sh
$ npm list vfile
extract-keywords-example@0.1.21
├─┬ to-vfile@7.0.0
│ └── vfile@5.0.1 
├─┬ unified@9.2.1
│ └── vfile@4.2.1 
└─┬ uniorg-parse@0.1.21
  └── vfile@4.2.1 
```

If I downgrade to `to-vfile@6` and use `import toVFile from 'to-vfile';`, the output is as follows:
```
<div><p>other org-mode
</p></div>
{}
```
(correct HTML, not correct keywords)

**Comment by wildwestrom** [comment]
If it helps, I'm using this as an npm package.

**Comment by wildwestrom** [comment]
How do you run tests? I have the repository cloned, that's all.

**Comment by rasendubi** [comment]
Found the root cause. See #14

**Comment by wildwestrom** [comment]
When can I expect this to reflect in npm so I can make use of this change?

**Comment by rasendubi** [comment]
Published the fix as v0.1.22

**Comment by wildwestrom** [comment]
Thank you so much!
